nix/nevow.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, isPy3k, twisted }:
+
+buildPythonPackage rec {
+  pname = "Nevow";
+  version = "0.14.4.dev";
+  name = "${pname}-${version}";
+  disabled = isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "twisted";
+    repo = "nevow";
+    rev = "aff713a6884a0ebf2214af5cfa3c1ea2d7d70669";
+    sha256 = "0wmfin9igj46dazyfkphk7jxqalvxisn2hjz79maqg4nbl0x3rcv";
+  };
+
+  propagatedBuildInputs = [ twisted ];
+
+  patches = [ ./nevow.patch ];
+
```

Comment:
Could we include a note here about the circumstances under which we could remove the patch? (e.g. is there a Nevow bug, or ...)

Reply:
Ah, indeed.  Actually, I asked for a new Nevow release and one was made yesterday so this will be slightly simplified to just point at that release and the whole patch becomes unnecessary.  I'll still leave a comment about why we need a newer Nevow.